### ISSUE TITLE
Termux Storage

### ISSUE THREAD

**Issue by Wjsfj51najdk** [Neo-Oli/chrooted-termux#4]
termux-setup-storage doesn't work
How to setup storage to copy downloaded deb files from sdcard and internal storage

**Comment by dyabbaljaki** [comment]
Link.

**Comment by ghost** [comment]
Append `command+=" -b /sdcard"` to start.sh on line 16

Or in simpler terms, use your text editor (Use nano if you don’t know what I am saying).
nano start.sh
and add  `command+=" -b /sdcard"` after
`command+=" -b /property_contexts"`

Example:

```
command+=" -b /property_contexts"
command+=" -b /sdcard
```

**Comment by ghost** [comment]
Oh and also I do not understand, sdcard?
You mean external storage? Not /sdcard or /storage/emulated/0?
Because that is not possible without root. (at this time)
Only possible is internal storage.

**Comment by luismene** [comment]
> termux-setup-storage doesn't work
> How to setup storage to copy downloaded deb files from sdcard and internal storage

**Comment by ghost** [comment]
Wow that is a long time ago, But I do not get what you mean.
